src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -10,37 +10,40 @@
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.motorcontrol.PWMSparkMax;
 import edu.wpi.first.wpilibj.motorcontrol.Spark;
+import edu.wpi.first.wpilibj.motorcontrol.VictorSP;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 public class Climber extends SubsystemBase
 {
   // HARDWARE //
-  private Spark spoolmotor;
-  private Spark spoolmotor2;
-  private Spark gearmotor;
+  private VictorSP spoolmotor;
+  private VictorSP spoolmotor2;
+  private VictorSP gearmotor;
 
   // Ports //
-  public static final int CLIMBER_SPARK_SPOOL = 5;
-  public static final int CLIMBER_SPARK_SPOOL_2 = 7;
+  public static final int CLIMBER_SPARK_SPOOL = 9;//5
+  public static final int CLIMBER_SPARK_SPOOL_2 = 8;//7
   public static final int CLIMBER_SPARK_GEAR = 6;
```

Comment:
Change these constant names cause we're not using sparks.  You can just make it CLIMBER_whatever